src/apprt/gtk-ng/class/application/GtkApplicationAction.zig

```diff
@@ -0,0 +1,57 @@
+//! GTK action handlers for Application
+const GtkApplicationAction = @This();
```

Comment:
I don't think we should put these in a separate file, take a look at my `gtk-ng-win` branch to see where I put them.

Reply:
OK, moved to `application.zig`